### PR TITLE
Implement the sign up UI

### LINKED ISSUE #4: Implement the Login UI
- [x] Login screen

Image

### LINKED ISSUE #8: Implement the SignUp UI
Image

### PR DESCRIPTION
**Pull request by truongtdph40209**
## Related issues
Resolved #8 

## Summary
Implement the sign up UI

### Steps:
1. Open `Index.js` and delete the code in render
2. **Paste** the following line:

```jsx
<SignUpTemplate/>
```
3. Run app

## Screenshots
| Light mode | Dark mode |
| --- | --- |
|image|image|

### CODE REVIEW

File: app/authentication/Signup.tsx
Comment (NguyenVanViet2004) [inline app/authentication/Signup.tsx:1]: `SignUp` not `Signup`

**Review by NguyenVanViet2004 — CHANGES_REQUESTED** [review]

**Review by NguyenVanViet2004 — CHANGES_REQUESTED** [review]
Rebase not merge

image

File: components/templates/SignUpTemplate.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import InputForm from '~/components/molecules/InputForm'
+import LinearGradientBackground from '~/components/molecules/LinearGradientBackground'
+import TextWithLink from '~/components/molecules/TextWithLink'
+import useTranslation from '~/hooks/useTranslation'
+
+const SignUpTemplate: React.FC = (): JSX.Element => {
+  const { t } = useTranslation()
+  const router = useRouter()
+
+  const ButtonSignUp = (): void => {
```
Comment (NguyenVanViet2004) [inline components/templates/SignUpTemplate.tsx:17]: Function's name???

**Review by NguyenVanViet2004 — CHANGES_REQUESTED** [review]
Delete constant SPACING_20 now

Ảnh màn hình 2024-10-02 lúc 08 48 05

**Review by KMTus — APPROVED** [review]

**Review by thanhdo29 — APPROVED** [review]

**Review by NguyenVanViet2004 — APPROVED** [review]

**Pull request merged**